README.md

```diff
@@ -11,4 +11,7 @@ Work in progress
 # Projects using ape
```

Comment:
Looks cool. Just make It Something like 

# Community Projects using ape

Reply:
done

Reply:
looks good